### PR TITLE
Update cri-o config to point to 4.8

### PR DESCRIPTION
**Pull request by umohnani8**
And add config and job for cri-o release-1.21

Signed-off-by: Urvashi Mohnani <umohnani@redhat.com>

### CODE REVIEW

**Comment by umohnani8** [comment]
cc @mrunalp @QiWang19

File: ci-operator/config/cri-o/cri-o/cri-o-cri-o-release-1.21.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    tag: "29"
+  machine-os-content-base:
+    name: "4.8"
+    namespace: ocp
+    tag: machine-os-content
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
```
Comment (mrunalp) [inline ci-operator/config/cri-o/cri-o/cri-o-cri-o-release-1.21.yaml:18]: Shouldn't we be using a newer golang?

Comment (umohnani8) [reply ci-operator/config/cri-o/cri-o/cri-o-cri-o-release-1.21.yaml]: Yup, other 4.8 jobs are using 1.14, bumped it up.

**Comment by umohnani8** [comment]
This is ready, gcp is having some issues right now. @mrunalp PTAL

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Pull request merged**